featuretools/primitives/standard/transform/natural_language/total_word_length.py

```diff
@@ -30,12 +33,11 @@ class TotalWordLength(TransformPrimitive):
 
     default_value = 0
 
-    def __init__(self, delimiters_regex=r"[- \[\].,!\?;\n]"):
-        self.delimiters_regex = delimiters_regex
+    def __init__(self, do_not_count=PUNCTUATION_AND_WHITESPACE):
```

Comment:
I think changing the parameter name means we'll need to bump the feature schema since old copies of this primitive that have a custom regex will no longer work.

Reply:
Oops, I meant to revert this change. I changed the behavior of this primitive to strip punctuation, and to split on the delimiters, so I think we can keep the parameter name.